client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,37 +2,65 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { find, endsWith, some, } from 'lodash';
+import url from 'url';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
 
+function isCandidateForFeature( media ) {
+	if ( ! media ) {
+		return false;
+	}
+
+	if ( media.mediaType === 'image' ) {
+		const image = media;
+
+		const imageIsTallEnough = 350 <= image.width || 350 <= image.naturalWidth;
```

Comment:
do we ever see `naturalWidth` with the changes I made to `wait-for-images-to-load`?